examples/sample.py

```diff
@@ -55,28 +56,42 @@
     output_vocab = checkpoint.output_vocab
 else:
     # Prepare dataset
-    dataset = Dataset.from_file(opt.train_path, src_max_len=50, tgt_max_len=50)
-    input_vocab = dataset.input_vocab
-    output_vocab = dataset.output_vocab
+    src = SourceField()
+    trg = TargetField()
```

Comment:
I just realized that I hard coded the filed names in 'sample.py' as 'src' and 'trg' and have been using them in the codes of the package.  I need to change that.  Will push more changes.